x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -406,6 +411,8 @@ Parser parameters control the parser operation and the types of error conditions
 |generate999Acks="false" |Generate 999 Implementation Acknowledgments, rather than 997 Functional Acknowledgments. By default, 997 Functional Acknowledgment transaction sets are generated for each received interchange. Changing this flag causes 999 Implementation Acknowledgments to be generated instead. Note that the support for 999 Implementation Acknowledgments does not include CTX segment generation.
 |reportSegmentErrors="true" |Report segment error details to sender in 997/999 flag. If true, the details of any segment errors are included in a generated 997/999. If false, the details are not included.
 |includeFASchema="true" |Expect 997 or 999 Functional Acknowledgments and include 997/999 schema. By default, the schema for 997/999 acknowledgment transaction sets is automatically included in the set of schemas used by the X12 EDI Module. If false, you need to directly specify the 997 and/or 999 schema if you want these to be processed as input. The schemas used for generating 997 or 999 Functional Acknowledgments are hard coded and cannot be modified.
+|acknowledgmentSchemaPath="" |Expect the path either from the file system or the classpath. If the path is specified, it overwrites the default acknowledgment.
```

Comment:
It overrides the default path?
How does a path override an acknowledgment?

Reply:
path